webapp/components/wpt-amend-metadata.js

```diff
@@ -59,7 +69,12 @@ class AmendMetadata extends LoadingState(ProductInfo(PolymerElement)) {
             <paper-input label="Bug URL" value="{{node.url}}" autofocus></paper-input>
           </div>
           <template is="dom-repeat" items="[[node.tests]]" as="test">
-            <li>[[test]]</li>
+            <li>
+              <div class="testname"> [[test]] </div>
+              <template is="dom-if" if="[[hasHref(node.product)]]">
+                <a href="[[getSearchURLHref(test)]]" target="_blank"> [Search in Monorail] </a>
```

Comment:
```suggestion
                <a href="[[getSearchURLHref(test)]]" target="_blank"> [Search on crbug] </a>
```

Not everyone knows what Monorail is.